### PR TITLE
minor fixups to the docs

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Updating the current parameters supported ans fix typos here and there

### CODE REVIEW

File: doc/maintaining_repo.md [hunk 1]
```diff
@@ -6,22 +6,22 @@ The Dockerfiles used in Offical Docker Hub repos for OSRF projects are hosted he
...
 
-> This can be done simply however by using the scripts described in accompanying docs to regenerate all Dockerfiles. For ROS, the scripts will use ros_buildfarm.common package to fetch the latest package index and append the proper version numbers.
+> This can be done simply however by using the scripts described in accompanying docs to regenerate all Dockerfiles. For ROS, the scripts will use `ros_buildfarm.common` package to fetch the latest package index and append the proper version numbers.
 
-Also, we now pin the version used for each ROS package. Note that however, we are using reprepro (a tool to handle local repositories of debian packages), and as in FAQ 3.1 is limited to one version per architectur. So from the Repeatability documentation referenced
+Also, we now pin the version used for each ROS package. Note that however, we are using reprepro (a tool to handle local repositories of debian packages), and as in FAQ 3.1 is limited to one version per architecture.
```
Comment (mikaelarguedas) [inline doc/maintaining_repo.md]: @ruffsl Not sure what the last sentence was trying to communicate (so I removed it). Feel free to push directly to this branch if you think a sentence should be added back

Comment (ruffsl) [reply doc/maintaining_repo.md]: Looks like a trailing sentence I forgot about.

I probably meant to say something along the line of repeatability like with what I commented/linked in  https://github.com/osrf/docker_images/issues/171#issuecomment-404290452 .

File: gazebo/create_dockerfiles.py [hunk 2]
```diff
@@ -24,19 +24,19 @@ def main(argv=sys.argv[1:]):
     parser.set()
     args = parser.parse(argv)
 
-    # Read platform perams
+    # Read platform params
```
Comment (mikaelarguedas) [inline gazebo/create_dockerfiles.py]: @ruffsl Maybe we should deduplicate these files as they seem to be identical between gazebo ros and ros2.
Do you expect these to diverge in the future or would it be safe to consolidate them ?

Comment (ruffsl) [reply gazebo/create_dockerfiles.py]: Originally they were each slightly different, but after refactoring the case specific logic into docker_templates lib, they should be the same. I think the CI makes the assumption that the scripts are executed in place, but we could probably just move `create_dockerfiles.py` `create_dockerfolders.py` and `create_dockerlibrary.py` to the root of the repo (e.g. `/scripts`) and just symlink them to each `ros`, `ros2`, `gazebo` folder.

**Pull request merged**